wpilibc/src/main/native/cpp/shuffleboard/ShuffleboardWidget.cpp

```diff
@@ -20,7 +20,7 @@ static constexpr const char* widgetStrings[] = {
     "ComboBox Chooser",
     "Split Button Chooser",
     "Encoder",
-    "Speed Controller",
+    "Motor Controller",
```

Comment:
To save other people the trouble, this matches what's already in shuffleboard.

Reply:
Specifically, it has already been changed in shuffleboard, so changing it here is correct. https://github.com/wpilibsuite/shuffleboard/blob/main/plugins/base/src/main/java/edu/wpi/first/shuffleboard/plugin/base/widget/SpeedControllerWidget.java#L31